app/lib/formatter.rb

```diff
@@ -14,8 +14,7 @@ def format(status)
 
     html = status.text
     html = encode(html)
-    html = simple_format(html, {}, sanitize: false)
-    html = html.delete("\n")
+    html = html.gsub(/(?:\n\r?|\r\n?)/, '<br />')
```

Comment:
some older operating systems use \r as a new line. probably overkill in this day and age I admit.